rts-core/src/components/turn_strategy.rs

```diff
@@ -0,0 +1,15 @@
+use crate::entity::game_actions::Action;
+use crate::entity::player::TurnStrategyRequester;
+use crate::exceptions::RtsException;
+
+pub enum TurnStrategy {
+    AI,
```

Comment:
I guess this is here that we will add a variant that runs python code to get an action from a game state?

Reply:
Exactly !